### PR TITLE
SSCS-9599: MYA – Split out document and audio/visual evidence uploads

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: locale/content.json [hunk 1]
```diff
@@ -1221,7 +1221,7 @@
       "location": "Lleoliad",
       "provideInformationOnline": "Darparu eich gwybodaeth ar-lein",
       "uploadLettersDocumentPhotos": "Uwchlwytho llythyrau, dogfennau neu luniau",
-      "uploadLettersDocumentPhotosAudioVideo": "Uwchlwytho llythyrau, dogfennau neu ffeiliau llun, sain neu fideo",
+      "uploadAudioVideo": "Upload audio or video files",
```
Comment (divisathyan) [inline locale/content.json:1224]: Shouldn't this be Welsh translation

**Review by divisathyan — COMMENTED** [review]
One comment

File: locale/content.json (hunk 1 above)
Comment (srochani) [reply locale/content.json:1224]: We have separate ticket (https://tools.hmcts.net/jira/browse/SSCS-9687)  for welsh translation

**Review by divisathyan — APPROVED** [review]
LG

**Comment by srochani** [comment]
Closing and reopening the PR

**Comment by srochani** [comment]
Closing and opening new PR

**Pull request merged**